istioctl/cmd/proxyconfig.go

```diff
@@ -61,6 +161,42 @@ func setupConfigdumpEnvoyConfigWriter(podName, podNamespace string, out io.Write
 	return cw, nil
 }
 
+func setupEnvoyLogConfig(param, podName, podNamespace string) (string, error) {
+	kubeClient, err := clientExecFactory(kubeconfig, configContext)
+	if err != nil {
+		return "", fmt.Errorf("failed to create k8s client: %v", err)
+	}
+	path := "logging"
+	if param != "" {
+		path = path + "?" + param
+	}
+	result, err := kubeClient.EnvoyDo(podName, podNamespace, "POST", path, nil)
+	if err != nil {
+		return "", fmt.Errorf("failed to execute command on envoy: %v", err)
```

Comment:
Envoy is a proper noun.  Capitalize as `Envoy`.

Reply:
Thanks.